bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ovh-angular-manager-navbar",
-  "version": "6.6.3",
+  "version": "6.6.4",
```

Comment:
Any reason to not use CDS for new release?

Reply:
No, is it so serious?

Reply:
No we should use CDS :)